projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/index.tsx

```diff
@@ -20,7 +21,7 @@ import './style.scss';
  */
 import type { Integration } from '../../../../types';
 
-const isMailPoetEnabled: boolean = !! window?.jpFormsBlocks?.defaults?.isMailPoetEnabled;
+const isMailPoetEnabled = Boolean( getScriptData()?.forms?.isMailPoetEnabled );
```

Comment:
An example of replacing use of jpFormsBlocks method with the getScriptsData() method for accessing our data.